rllib/utils/tests/test_serialization.py

```diff
@@ -0,0 +1,109 @@
+import gym
+import numpy as np
+import unittest
+
+from ray.rllib.utils.serialization import (
+    gym_space_from_dict,
+    gym_space_to_dict,
+)
+
+
+def _assert_array_equal(eq, a1, a2, margin=None):
+    if margin:
+        for a in zip(a1, a2):
+            eq(a[0], a[1], margin)
+    else:
```

Comment:
This works with `margin=None`, which you wisely put as default, so I think you can simplify this to just

```
for a in zip(a1, a2):
            eq(a[0], a[1], margin)
```

Reply:
🤯 had no idea. thanks.